pkg/registry/batch/cronjob/strategy_test.go

```diff
@@ -92,6 +93,13 @@ func TestCronJobStrategy(t *testing.T) {
 	if len(errs) == 0 {
 		t.Errorf("Expected a validation error")
 	}
+
+	// Make sure we correctly implement the interface.
+	// Otherwise a typo could silently change the default.
+	var gcds rest.GarbageCollectionDeleteStrategy = Strategy
```

Comment:
Ok I've added it to my list and will change it across all controllers in a followup PR.